openjdk9/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,57 @@
+# Copyright 2014 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM ${docker.appengine.image}
+ENV DEBIAN_FRONTEND noninteractive
+
+# Create env vars to identify image
+ENV OPENJDK_VERSION ${openjdk.version}
+ENV GAE_IMAGE_NAME ${project.artifactId}
+ENV GAE_IMAGE_LABEL ${docker.tag.long}
+
+RUN \
+ # add debian 'sid' repo in order to install openjdk-9
+ echo 'deb http://httpredir.debian.org/debian buster main' > /etc/apt/sources.list.d/buster.list \
+ # add debian jessie, jessie-backports repo in order to install downgraded libcups2 dependency
+ && echo 'deb http://httpredir.debian.org/debian jessie main' > /etc/apt/sources.list.d/jessie.list \
+ && echo 'deb http://httpredir.debian.org/debian jessie-backports main' > /etc/apt/sources.list.d/jessie-backports.list \
+
+ && apt-get -q update \
+ && apt-get -y -q --no-install-recommends install \
+    # jdk and its dependencies
+    ca-certificates-java=20170531'*' \
+    openjdk-9-jdk-headless=${openjdk.version}'*' \
+    libcups2=2.2.1-8~bpo8+1 \
```

Comment:
can we put in a comment above or here, why we need to downgrade libcups2? 
I would even double check if we can remove it completely! (unless of course we want to explicitly support printing from cloud deployed apps or rather the JDK has some weird dependency on CUPS)